tests/e2e/tidbcluster/serial.go

```diff
@@ -160,18 +160,19 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 
 		ginkgo.It("Scaling tidb cluster with advanced statefulset", func() {
```

Comment:
in this case, sts `webhook` is enabled. you must support webhook, otherwise, this test can't pass.